server/compose/envoy/yaml_decode.go

```diff
@@ -103,6 +103,40 @@ func (d *auxYamlDoc) unmarshalPageBlocksNode(r *types.Page, n *yaml.Node) (refs
 	return
 }
 
+func (d *auxYamlDoc) unmarshalNamespaceBlocksNode(r *types.Namespace, n *yaml.Node) (refs map[string]envoyx.Ref, idents envoyx.Identifiers, err error) {
```

Comment:
I'm not a fan of this copy paste; since these and page blocks are basically the same; can we reuse the code?